build.proj

```diff
@@ -7,9 +7,10 @@
     <BuildManaged Condition="'$(BuildManaged)' == ''">true</BuildManaged>
     <BuildPackages Condition="'$(BuildPackages)' == '' and '$(DirectoryToBuild)' == ''">true</BuildPackages>
 
-    <BuildDependsOn Condition="'$(BuildNative)' == 'true'">$(BuildDependsOn);BuildNative</BuildDependsOn>
-    <BuildDependsOn Condition="'$(BuildManaged)' == 'true'">$(BuildDependsOn);BuildManaged</BuildDependsOn>
-    <BuildDependsOn Condition="'$(BuildPackages)' == 'true'">$(BuildDependsOn);Pack</BuildDependsOn>
+    <BuildDependsOn Condition="'$(BuildNative)' == 'true' and '$(BuildTests)' != 'only'">$(BuildDependsOn);BuildNative</BuildDependsOn>
```

Comment:
What are the switch values for `BuildTests` from looking at this changes I suppose it passed down by arcade.

Reply:
These switch values are defined in build.ps1 and build.sh.

Reply:
The possible values are: '', 'true', 'only'.